proxy/src/main/java/com/wavefront/agent/ProxyConfigDef.java

```diff
@@ -1514,4 +1514,37 @@ public abstract class ProxyConfigDef extends Configuration {
 
   boolean enableHyperlogsConvergedCsp = false;
   boolean receivedLogServerDetails = true;
+
+  /** CSP console URL. This will be used in many places like getting token. */
+  public static final String cspBaseUrl = "https://console-stg.cloud.vmware.com";
+
+  @Parameter(
+      names = {"--csp-api-token"},
```

Comment:
@laullon i just happened to check this PR out as our team was waiting on the proxy release to support csp auth in the opApps k8s operator. Quick question, IIUC, this would translate to a user config option in the wavefront.conf file. In that case, to maintain consistency with other proxy config options,  this can be named in camel case. Ex: "--cspApiToken" . 
Same comment for other newly added csp specific config parameters.

Reply:
sure. no problem